src/content/zh-tw/_project.yaml

```diff
@@ -0,0 +1,22 @@
+---
+announcement:
+  description: </#cdata-section></style><style><#cdata-section> 錯過了今年的 <b>Chrome 开发峰会</b>?
```

Comment:
Contents inside `<style>` tag is missing
@sotayamashita

Reply:
@agektmr I’ll keep you posted when we figure out how to handle the Boolean type. I will overwrite the value directly until the issue will be solved. gitlocalize/feedback#28